package.json

```diff
@@ -6,13 +6,22 @@
   "main": "app.js",
   "type": "module",
   "author": "@itznotabug",
+  "scripts": {
+    "dev": "nodemon -e ejs app.js",
```

Comment:
I think `nodemon` should watch other files too (eg: `js`), except `json` maybe as those are used for data source & can change when changing settings config causing unexpected restarts.